lit-element/package.json

```diff
@@ -22,7 +22,7 @@
         "test": "jest"
     },
     "dependencies": {
-        "@agentx/agentx-js-api": "^1.1.3"
+        "@agentx/agentx-js-api": "^1.2.0"
```

Comment:
wonder why these files show as changed... do you need to update this fork?

Reply:
I can do so

Reply:
DONE